app/resources/configuration_resource.rb

```diff
@@ -0,0 +1,18 @@
+require 'okapi'
+
+class ConfigurationResource < JSONAPI::Resource
+  attributes :customer_id, :api_key
+
+  def self.find_by_key(key, options = {})
```

Comment:
We need some documentation around this method (i.e. it's necessary to implement singular, not active-record based resources), preferably a link to the issue on github that outlines this solution.